diff --git a/node_modules/@perses-dev/core/dist/cjs/schema/panel.js b/node_modules/@perses-dev/core/dist/cjs/schema/panel.js
index 30f3a7e..e823811 100644
--- a/node_modules/@perses-dev/core/dist/cjs/schema/panel.js
+++ b/node_modules/@perses-dev/core/dist/cjs/schema/panel.js
@@ -58,7 +58,8 @@ const panelDisplaySpec = _zod.z.object({
 const querySpecSchema = _zod.z.object({
     kind: _zod.z.string().min(1),
     spec: _zod.z.object({
-        plugin: _plugin.pluginSchema
+        plugin: _plugin.pluginSchema,
+        hidden: _zod.z.boolean().optional()
     })
 });
 const linkSchema = _zod.z.object({
diff --git a/node_modules/@perses-dev/core/dist/model/query.d.ts b/node_modules/@perses-dev/core/dist/model/query.d.ts
index dd044e4..d365a26 100644
--- a/node_modules/@perses-dev/core/dist/model/query.d.ts
+++ b/node_modules/@perses-dev/core/dist/model/query.d.ts
@@ -5,6 +5,7 @@ import { ProfileData } from './profile-data';
 import { LogData } from './log-data';
 interface QuerySpec<PluginSpec> {
     plugin: Definition<PluginSpec>;
+    hidden?: boolean;
 }
 /**
  * A generic query definition interface that can be extended to support more than just TimeSeriesQuery
diff --git a/node_modules/@perses-dev/core/dist/model/time-series-data.d.ts b/node_modules/@perses-dev/core/dist/model/time-series-data.d.ts
index a5450d8..bd764d9 100644
--- a/node_modules/@perses-dev/core/dist/model/time-series-data.d.ts
+++ b/node_modules/@perses-dev/core/dist/model/time-series-data.d.ts
@@ -21,6 +21,7 @@ export interface TimeSeries {
     labels?: Labels;
 }
 export interface TimeSeriesMetadata extends BaseMetadata {
+    isSelectable?: boolean;
     [key: string]: unknown;
 }
 //# sourceMappingURL=time-series-data.d.ts.map
\ No newline at end of file
diff --git a/node_modules/@perses-dev/core/dist/model/variables.d.ts b/node_modules/@perses-dev/core/dist/model/variables.d.ts
index d84ccd7..73638e7 100644
--- a/node_modules/@perses-dev/core/dist/model/variables.d.ts
+++ b/node_modules/@perses-dev/core/dist/model/variables.d.ts
@@ -35,6 +35,7 @@ export interface BuiltinVariableDefinition extends Definition<BuiltinVariableSpe
 export interface BuiltinVariableSpec extends VariableSpec {
     value: () => string;
     source: string;
+    sourceDescription?: string;
 }
 export type VariableDefinition = TextVariableDefinition | ListVariableDefinition;
 /**
diff --git a/node_modules/@perses-dev/core/dist/schema/panel.js b/node_modules/@perses-dev/core/dist/schema/panel.js
index 6be0def..154aceb 100644
--- a/node_modules/@perses-dev/core/dist/schema/panel.js
+++ b/node_modules/@perses-dev/core/dist/schema/panel.js
@@ -19,7 +19,8 @@ export const panelDisplaySpec = z.object({
 export const querySpecSchema = z.object({
     kind: z.string().min(1),
     spec: z.object({
-        plugin: pluginSchema
+        plugin: pluginSchema,
+        hidden: z.boolean().optional()
     })
 });
 export const linkSchema = z.object({
